js/map.js

```diff
@@ -54,13 +59,42 @@ function generateMonsters(){
     monsters = [];
     let numMonsters = level+1;
     for(let i=0;i<numMonsters;i++){
-        spawnMonster();
+        spawnMonster(level);
     }
 }
 
-function spawnMonster(){
+function spawnMonster(level){
     // let monsterType = shuffle([Goose, Ant, Mushroom, Eater, Toast])[0];
-    let monsterType = shuffle([Rock, Paper, Scissors ,Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti])[0];
+
+     let monsterType = shuffle(chooseMonsterType(level))[0];
     let monster = new monsterType(randomPassableTile());
     monsters.push(monster);
 }
+
+function chooseMonsterType(level){  
+
+  if(level < 4){    
+    return normalMonster();
+  }
+  else if(level < 7){
```

Comment:
at the moment, I believe the game is programmed to reset at level 6. I'll make a new issue to bump it up to 10.

Reply:
Sounds good.